src/freenet/clients/http/QueueToadlet.java

```diff
@@ -44,10 +45,11 @@
 import freenet.client.async.PersistentJob;
 import freenet.client.async.TooManyFilesInsertException;
 import freenet.client.filter.ContentFilter;
-import freenet.client.filter.KnownUnsafeContentTypeException;
 import freenet.client.filter.FilterMIMEType;
+import freenet.client.filter.KnownUnsafeContentTypeException;
```

Comment:
This should not have included reorganizing imports - it contains no formatting commits.